console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/widget/KapuaPagingToolBar.java

```diff
@@ -19,8 +19,17 @@
 
     private static final ConsoleMessages MSGS = GWT.create(ConsoleMessages.class);
 
+    /**
+     * Max length of user input field for page number.
+     */
+    public static final String MAX_TEXT_LEN = "4";
```

Comment:
Wouldn't it be safer to use an `Integer` here, and use its `toString()` in the `setAttibute()` later on?

Reply:
Shure, I didn't know from which side to go.

Reply:
Well, at least using `Integer` you can be certain that only an integer will be written in the variable :smile: